eth/utils/rlp.py

```diff
@@ -11,9 +16,15 @@
     ValidationError,
 )
 
+from eth.rlp.blocks import (
+    BaseBlock,
+)
+
+DiffObjectType = Optional[Iterable[Tuple[str, str, str]]]
+
 
 @to_tuple
-def diff_rlp_object(left, right):
+def diff_rlp_object(left: BaseBlock, right: BaseBlock) -> DiffObjectType:
```

Comment:
Since this is only being used once, I'd prefer to use the type in it longer form and just reformat as:

```
def diff_rlp_object(left: BaseBlock,
                             right: BaseBlock) -> Optional[Iterable[Tuple[str, str, str]]]
...
```